src/RequestSender.ts

```diff
@@ -287,7 +287,7 @@ export class RequestSender {
 
   // Max retries can be set on a per request basis. Favor those over the global setting
   _getMaxNetworkRetries(settings: RequestSettings = {}): number {
-    return settings.maxNetworkRetries &&
+    return settings.maxNetworkRetries !== undefined &&
```

Comment:
Do we need this? I thought you said `Number.isInteger` is enough.

Reply:
Typescript wants it.